src/Form/MetadataDisplayForm.php

```diff
@@ -174,17 +174,41 @@ public function save(array $form, FormStateInterface $form_state) {
     return $status;
   }
 
+  public static function flattenKeys(array $array, string $recursive_key = '', string $recursive_type = '') {
+    $return = [];
+    if (is_array($array)) {
```

Comment:
This is unneeded, the Function signature will not let you pass a non array

Reply:
Resolved via b80eb4ae69f1b7b8f6b8723a6a4c0eeeffa42de5.